.github/workflows/test.yaml

```diff
@@ -135,6 +148,12 @@ jobs:
           cmake --build build --target ech-test > ech-test.log
           grep 'sni=encrypted' ech-test.log
 
+      - name: Run PQ connect test
```

Comment:
This will need to be reconciled with https://github.com/rustls/rustls-ffi/pull/517 depending on merge order.

Reply:
Done.